### PR TITLE
S02 us30 create video

### PR DESCRIPTION
**Pull request by camille-haxa**

### CODE REVIEW

**Review by aurelienLavanchy — APPROVED** [review]

**Review by JohaneDecamps — APPROVED** [review]

File: server/database/models/VideoRepository.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    return row[0];
+  }
+
+  // Edit
+
+  async edit(video) {
+    const { is_premium, is_free, requires_account, category_id } = video;
+
+    const [result] = await this.database.query(
+      `UPDATE ${this.table} SET is_premium=?, is_free=?, requires_account=?, category_id=?`,
```
Comment (MichaelLELU) [inline server/database/models/VideoRepository.js:61]: WHERE  ?

Comment (aurelienLavanchy) [inline server/database/models/VideoRepository.js:61]: Missing `WHERE id=?`

**Review by aurelienLavanchy — CHANGES_REQUESTED** [review]

File: server/database/models/VideoRepository.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      [is_premium, is_free, requires_account, category_id]
+    );
+
+    return result;
```
Comment (MichaelLELU) [inline server/database/models/VideoRepository.js:65]: it missing .affectedRows

Comment (camille-haxa) [reply server/database/models/VideoRepository.js]: done

File: server/database/models/VideoRepository.js (hunk 1 above)
Comment (camille-haxa) [reply server/database/models/VideoRepository.js]: done

**Review by aurelienLavanchy — APPROVED** [review]

**Review by ioayoub — APPROVED** [review]

**Review by MichaelLELU — APPROVED** [review]

**Review by JohaneDecamps — APPROVED** [review]

**Pull request merged**